packages/gestalt/src/TileData.js

```diff
@@ -0,0 +1,104 @@
+// @flow strict
+import { type Node } from 'react';
+import DataPoint from './Datapoint.js';
+import { useColorScheme } from './contexts/ColorSchemeProvider.js';
+import { type DataPointBaseProps } from './Datapoint/DataPointBaseProps.js';
+import Tile from './Tile/Tile.js';
+
+// Note: 03 is ommited because it doesn't have a corresponding dark token
+type DataVisualizationColors =
+  | '01'
+  | '02'
+  | '04'
+  | '05'
+  | '06'
+  | '07'
+  | '08'
+  | '09'
+  | '10'
+  | '11'
+  | '12';
+
+type Props = {|
+  ...DataPointBaseProps,
+  /**
+   * disables component interactivity
+   */
+  disabled?: boolean,
+  /**
+   * An identifier to be passed in a callback, and distinguish multiple DataPoints
+   */
+  id?: string,
+  /**
+   * Handler if the item is selected
+   */
+  onChange: ({| selected: boolean, id?: string |}) => void,
+  /**
+   * Controls whether the tile is selected or not
+   */
+  selected?: boolean,
+  /**
+   * A valid color code from the data visualization palette
+   */
+  color?: DataVisualizationColors,
+  /**
+   * Shows a checkbox. Useful when multi-select is available
+   */
+  showCheckbox?: boolean,
+  /**
+   * Adds a Tooltip on hover/focus of the Tile. See the with [Tooltip](#Tooltip) variant to learn more.
+   */
+  tooltip?: string,
```

Comment:
Check out how we add tooltips on other components — there are other controls that will be needed, so this will need to be an object. Badge and IconButton are good examples of this (Datapoint is very much not!)